tools/flowc/backends/nim/natives/string/getCharAt.nim

```diff
@@ -1,13 +1,7 @@
 import unicode
 
 func $F_0(getCharAt)*(s: RtString, i: int32): RtString =
-  when use16BitString:
-    if i >= 0 and i < rt_string_len(s):
-      return @[s[i]]
-    else:
-      return rt_empty_string()
+  if i >= 0 and i < rt_string_len(s):
+    return when use16BitString: @[s[i]] else: unicode.toUTF8(unicode.runeAt(s, i)) 
```

Comment:
It still seems wrong to use "rune" for the uft8 case. But Maybe we should just focus on utf16 first and get that working.

Reply:
ok, I switched to `return when use16BitString: @[s[i]] else: $s[i]`